e2core/server/server.go

```diff
@@ -75,10 +76,14 @@ func New(sync *syncer.Syncer, opts *options.Options) (*Server, error) {
 
 	router.WithMiddlewares(server.openTelemetryMiddleware())
 	router.WithMiddlewares(scopeMiddleware)
+	if opts.AdminEnabled() {
+		router.POST("/name/:ident/:namespace/:name", auth.AuthorizationMiddleware(opts, server.executePluginByNameHandler()))
```

Comment:
How are we protecting against ident collisions across tenants in multi-tenant environments?

Reply:
The AuthorizationMiddleware is sort of a bad name because in addition to authorizing your request it resolves your friendly tenant name into its globally unique identifier. 

This is actually what happened before as well but it was sort of silly to do {UUID}.{UUID}/{namespace}/... since the tenant id is already globally unique.

Reply:
Below is an example of what e2core see with the tenant resolution and authorization enabled.; the builder/e2core SDK mismatch is a WIP item. 

```
starting fqmn://93fad26e-e691-4f2d-9fd1-b797c369b91c/default/myFunc@1e754146dc13b7a8502acf7f7d55f5d697e344d0cc7df2e5a33974b4a886de35-1fff8a ...","timestamp":"2023-01-19T10:39:10.766181423Z","level":3,"app":{"name":"fqmn://93fad26e-e691-4f2d-9fd1-b797c369b91c/default/myFunc@1e754146dc13b7a8502acf7f7d55f5d697e344d0cc7df2e5a33974b4a886de35-1fff8a"}}
2023-01-19 05:39:10 {"log_message":"(E) failed to start fqmn://93fad26e-e691-4f2d-9fd1-b797c369b91c/default/myFunc@1e754146dc13b7a8502acf7f7d55f5d697e344d0cc7df2e5a33974b4a886de35 worker: failed to reconcilePoolSize: failed to addThread more than numRetries: runnable returned OnChange error: failed to addInstance: failed to builder.New: failed to linker.Instantiate: unknown import: `env::cache_get` has not been defined","timestamp":"2023-01-19T10:39:10.91296559Z","level":1}
```